apps/Dataman/src/components/Button.tsx

```diff
@@ -0,0 +1,14 @@
+import { Button, View } from "@ant-design/react-native";
+
+export const FormButton = () => {
+  return (
+    <View style={{ alignItems: "center", marginVertical: 10 }}>
```

Comment:
why not tailwind, same issue in various other places

Reply:
Issue resolved